### ISSUE TITLE
JAWS doesn't narrates element role inside group with aria-roledescription

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluentui#14110]
Based on the bug:

JAWS doesn't narrates element role inside group with aria-roledescription #407 
https://github.com/FreedomScientific/VFO-standards-support/issues/407 

which I reported to the JAWS we need to reflect this issue in our cards component. 

Suggestion:
as we cannot rely on the JAWS will fix it soon, workaround is remove "aria-roledescription" attribute from card and move the narration of it into the label (aria-label, aria-labelledby, aria-describedby)
